src/picknik_ur_base_config/config/control/picknik_ur.ros2_control.yaml

```diff
@@ -14,7 +14,7 @@ controller_manager:
     robotiq_activation_controller:
       type: robotiq_controllers/RobotiqActivationController
     streaming_controller:
-      type: position_controllers/JointGroupPositionController
+      type: joint_trajectory_controller/JointTrajectoryController
```

Comment:
I think calling a JTC-type controller a streaming controller is confusing. If we need this for servo, you should rename the controller completely. Can't you use `joint_trajectory_controller` or does it one have a specific config?